database/src/main/resources/db/migration/V1.71.0__RHCLOUD-22470_insert_test_action_events_bundle_application_event_type.sql

```diff
@@ -0,0 +1,42 @@
+-- Insert a bundle, an application and an event type that will be specifically
+-- used for test actions and events.
+
+INSERT INTO bundles(
```

Comment:
Instead of creating a new BAET hierarchy for this, we could add a new event type under `[bundle=console, app=notifications]`. This event type could be:
- name: `integration-test`
- display_name: `Integration Test`

Reply:
All right! I am assuming that you meant `app=integrations`, right?